src/table/table.scss

```diff
@@ -99,6 +103,7 @@
     visibility: hidden;
     @include themed {
       fill: t(iui-icons-color-actionable);
+      opacity: t(iui-opacity-3);
```

Comment:
Another option could be to use `iui-icons-color` for the hover state & `iui-icons-color-actionable` for the active sort state.

Reply:
Done